<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>602</height>
   </rect>
  </property>
  <property name="floating">
   <bool>false</bool>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>&amp;GeoGig Navigator</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="filterWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QComboBox" name="comboEndpoint"/>
          </item>
          <item>
           <widget class="QToolButton" name="btnAddServer">
            <property name="toolTip">
             <string>Add GeoGig server</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnEditServer">
            <property name="toolTip">
             <string>Edit GeoGig server</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnDeleteServer">
            <property name="toolTip">
             <string>Delete GeoGig server</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnRefresh">
            <property name="toolTip">
             <string>Refresh</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnAddRepo">
            <property name="toolTip">
             <string>Add new repository</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTreeWidget" name="repoTree">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="headerStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionAddGeoGigServer">
   <property name="text">
    <string>Add GeoGig server</string>
   </property>
   <property name="toolTip">
    <string>Add GeoGig server</string>
   </property>
  </action>
  <action name="actionShowFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Filter</string>
   </property>
   <property name="toolTip">
    <string>Show Filter</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionAddLayer">
   <property name="text">
    <string>Add layer to repository...</string>
   </property>
   <property name="toolTip">
    <string>Add layer to repository...</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove selected element</string>
   </property>
  </action>
  <action name="actionCreateRepository">
   <property name="text">
    <string>Create new repository</string>
   </property>
   <property name="toolTip">
    <string>Create new repository</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Help</string>
   </property>
  </action>
  <action name="actionManageRemotes">
   <property name="text">
    <string>Manage remotes</string>
   </property>
   <property name="toolTip">
    <string>Manage remotes</string>
   </property>
  </action>
  <action name="actionPull">
   <property name="text">
    <string>Pull</string>
   </property>
   <property name="toolTip">
    <string>Pull from remote</string>
   </property>
  </action>
  <action name="actionPush">
   <property name="text">
    <string>Push</string>
   </property>
   <property name="toolTip">
    <string>Push to remote</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
